pkg/ebpf/tracee.go

```diff
@@ -864,6 +905,16 @@ func (t *Tracee) attachProbes() error {
 		if !ok {
 			continue
 		}
+		if event.Syscall {
+			err := t.probes.Attach(probes.SyscallEnter__Internal)
```

Comment:
I think its ok to call this over and over (as the implementations allow that, checking if link is attached before trying to attacha gain).